### PR TITLE
fix: Table onChange event is triggered twice

### LINKED ISSUE #25434: Table 异步函数会被触发两次，当使用 size changer 的时候
- [ ] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### Reproduction link
[https://riddle.alibaba-inc.com/riddles/b1af792f](https://riddle.alibaba-inc.com/riddles/b1af792f)

### Steps to reproduce
官方示例：https://ant.design/components/table-cn/#components-table-demo-ajax

点击页码切换器，onChange 会被触发两次。

### What is expected?
只被触发一次

### What is actually happening?
被触发了两次

| Environment | Info |
|---|---|
| antd | 4.4.0 |
| React | 16.8  |
| System | mac |
| Browser | chrome |

### PR DESCRIPTION
**Pull request by zhangchen915**
### 🤔 This is a ...

- [x] Bug fix

### 🔗 Related issue link

close #25434 

### 💡 Background and solution

合并了 Table 组件 usePagination hook 中的 onInternalShowSizeChange  和 onInternalChange 函数

### 📝 Changelog
Documentation update, no code changes.

Language | Changelog
-- | --
🇺🇸 English |  Fix Table `onChange` triggered multiple times when change page size.
🇨🇳 Chinese | 修复 Table 切换页条目数时 `onChange` 触发多次的问题。

### ☑️ Self Check before Merge

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by 07akioni** [comment]
Can you add a test case directly corresponds to the issue #25434?

**Comment by xrkffgg** [comment]
PR 里的 changelog 写一下

File: components/table/hooks/usePagination.ts [hunk 1]
```diff
@@ -101,8 +101,6 @@ export default function usePagination(
       pageSize,
     });
 
-    onChange(1, pageSize);
```
Comment (AlbertAZ1992) [inline components/table/hooks/usePagination.ts:104]: It seems onChange should not be remove ?
setInnerPagination has set current to 1.

**Comment by zhangchen915** [comment]
已经补充 Changelog 和测试用例

**Comment by hengkx** [comment]
这个是PR是改模版了？

**Comment by yoyo837** [comment]
> 这个是PR是改模版了？

删了不选不填的部分

**Comment by hengkx** [comment]
Changelog 不是中英文的表格吗 这样还能识别吗

**Comment by yoyo837** [comment]
> Changelog 不是中英文的表格吗 这样还能识别吗

应该不能😂

**Pull request merged**